app/view/show.py

```diff
@@ -50,9 +50,11 @@ def get_bug_data(cves, pkgs, group):
     }
 
     task_severity = severitiy_mapping.get(group.severity.name)
+    # project id 5 for [community], 1 for [extra]/[core]
+    project = 5 if all([pkg.database == 'community' for pkg in versions]) else 1
```

Comment:
this will fail if there is any update in [community-testing] plus i believe this also needs to handle [multilib] and [multilib-testing]. Maybe its worth giving it a own function to resolve the project id?